### PR TITLE
Add TPC-DS DDL test cases

### PR DESCRIPTION
**Pull request by yaooqinn**
yaooqinn [Closes #487](https://github.com/yaooqinn/kyuubi/pull/487)    Target Issue [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_

Verifying TPC-DS DDLs, we should cover DML, DQL later, and maybe benchmark tests

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by pan3793 — APPROVED** [review]

**Review by turboFei — APPROVED** [review]

File: kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala [hunk 1]
```diff
@@ -0,0 +1,670 @@
...
+      Field("sr_item_sk", int),
+      Field("sr_customer_sk", int),
+      Field("sr_cdemo_sk", int),
+      Field("sr_hdemo_sk", int),
+      Field("sr_addr_sk", int),
+      Field("sr_store_sk", int),
+      Field("sr_reason_sk", int),
+      Field("sr_ticket_number", int),
+      Field("sr_return_quantity", int),
+      Field("sr_return_amt", int),
```
Comment (ulysses-you) [inline kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala:117]: Spark use the `long` type to define these field type and some other, do we need to follow those ? Althought the standard tpcds seems does not require the type of `identifier` which allowed both `int` and `long`.

Comment (yaooqinn) [reply kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala:117]: FYI, https://github.com/apache/spark/pull/32037. I am going to change Spark's definition. It's no reason to use long for those fields

Comment (ulysses-you) [reply kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala:117]: ah I see, let's wait that PR merged first

Comment (yaooqinn) [reply kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala:117]: the spark pr merged, can you proceed?

**Comment by ulysses-you** [comment]
thanks merging to master for v1.2.0

**Pull request closed without merging**